### PR TITLE
feat: DAMD-137 (proper motion and parallax in pfsDesign and pfsConfig)

### PR DESCRIPTION
**Pull request by monodera**
- Add `epoch`, `pmRa`, `pmDec`, and `parallax` for targets in the DESIGN table of pfsDesign and pfsConfig with the same formats as those in the GUIDESTAR table.
- Modify `datamodel.txt` and `tests_PfsConfig.py` accordingly.

### CODE REVIEW

**Review by PaulPrice — CHANGES_REQUESTED** [review]
This looks reasonable, but there need to be some changes before merging. In particular, `test_PfsFluxReference.py` fails due to this change, and I'm sure there must be similar failures in drp_stella.

File: datamodel.txt [hunk 1]
```diff
@@ -385,6 +385,10 @@ The DESIGN table lists for each object:
       targetType     32-bit int (enumerated type: e.g. SCIENCE,SKY,FLUXSTD)
       fiberStatus    32-bit int (enumerated type: e.g. GOOD,BROKENFIBER,BLOCKED,BLACKSPOT)
       pfiNominal     pair of 32-bit floats (millimeters on the PFI)
+      epoch          7-digit string
```
[the remark this answers is not in the document]
Comment (monodera) [reply datamodel.txt:388]: I copied it from the explanation in the guide star table which was written by someone before. It's something like "J2000.0".

[the remark this answers is not in the document]
Comment (PaulPrice) [reply datamodel.txt:388]: I think "7-character string" would be more accurate. "Digit" implies solely numbers.

File: python/pfs/datamodel/pfsConfig.py [hunk 2]
```diff
@@ -176,6 +186,10 @@ class PfsDesign:
                "catId": "J",
                "objId": "K",
                "targetType": "J",
+               "epoch": "A",
+               "pmRa": "D",
+               "pmDec": "D",
```
[the remark this answers is not in the document]
Comment (monodera) [reply python/pfs/datamodel/pfsConfig.py:191]: I'll change it to "E".

I actually copied from `guideStars.py` and it also has apparently inconsistent definitions (at lines around 138).  

```
            Column("pmRa", "D", array=self.pmRa, unit='mas/year'),
            Column("pmDec", "D", array=self.pmDec, unit='mas/year'),
            Column("parallax", "E", array=self.parallax, unit='mas'),
```

[the remark this answers is not in the document]
Comment (PaulPrice) [reply python/pfs/datamodel/pfsConfig.py:191]: Would you fix those (in a separate commit) while you're in there, please?

[the remark this answers is not in the document]
Comment (monodera) [reply python/pfs/datamodel/pfsConfig.py]: done

**Comment by monodera** [comment]
I fixed `test_PfsFluxReference.py`, but not those for `drp_stella`.

**Comment by PaulPrice** [comment]
They will need to be fixed before merging.

**Comment by PaulPrice** [comment]
The best way would be to run the tests (`setup -jr . ` and then `scons`) and fix anything that fails. Alternatively, you could `grep` for the `PfsConfig` constructor.

**Comment by monodera** [comment]
I did the test and made modifications accordingly. The corresponding PRs have been made in `drp_stella`.

**Pull request merged**